packages/block-library/src/gallery/shared.js

```diff
@@ -21,3 +28,20 @@ export const pickRelevantMediaFiles = ( image, sizeSlug = 'large' ) => {
 	}
 	return imageProps;
 };
+
+export function isGalleryV2Enabled() {
+	if ( typeof process !== 'undefined' && process.env?.NODE_ENV === 'test' ) {
+		return true;
+	}
+
+	if ( Platform.isWeb && window.wp.galleryBlockV2Enabled ) {
+		return true;
+	}
+
+	const settings = select( blockEditorStore ).getSettings();
+	if ( settings.__unstableGalleryWithImageBlocks ) {
+		return true;
+	}
+
+	return false;
```

Comment:
NP: at this point, we can just:

```
return settings.__unstableGalleryWithImageBlocks
```

i.e. without the conditional.

Reply:
Doh! - yeh that makes more sense - fixed.